jupyterhub/jupyterhub_config.py

```diff
@@ -9,29 +9,36 @@
 c.JupyterHub.admin_access = True
 c.JupyterHub.hub_ip = os.environ['HUB_IP']
 
-## Authentication
+
+# Authentication
 # These settings are specific to the CRC's proprietary authentication package `crc_jupyter_auth`
-# c.JupyterHub.authenticator_class = 'crc_jupyter_auth.RemoteUserAuthenticator'
-# c.Authenticator.required_vpn_role = 'SAM-SSLVPNSAMUsers'
-# c.Authenticator.missing_user_redirect = 'https://crc.pitt.edu/Access-CRC-Web-Portals'
-# c.Authenticator.missing_role_redirect = 'https://crc.pitt.edu/Access-CRC-Web-Portals'
-# c.Authenticator.admin_users = {'yak73', 'leb140'}
-c.Authenticator.debug = True
+c.JupyterHub.authenticator_class = 'crc_jupyter_auth.RemoteUserAuthenticator'
```

Comment:
In the future we should move these values into a .env file that is loaded into the environment by the docker-compose recipe. This will make settings management easier in the future, especially whenever we get around to running things on Kubernetes.

Reply:
Okay, sounds great!